Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -2780,7 +2829,7 @@ def setup_(self, step, stepName, stepDict, k, properties):
             dd4hepGeom+=stepDict[step][k]['--geometry']
             stepDict[stepName][k] = merge([{'--geometry' : dd4hepGeom, '--procModifiers': 'dd4hep'}, stepDict[step][k]])
     def condition(self, fragment, stepList, key, hasHarvest):
-        return ('2021' in key or '2023' in key or '2026' in key) and ('FS' not in key)
+        return fragment=="TTbar_14TeV" and ('2021' in key or '2023' in key or '2026' in key) and ('FS' not in key)
```

Comment:
Why limiting DD4Hep only to TTbar? We just produced a full set of samples to validate DD4Hep in pre6 and we wouldn't have been able to do that with this change.

Reply:
Typically, the special workflows have been used as examples/tests rather than populating the entire matrix. This workflow in particular involves only a few changed flags in the cmsDriver commands, so relvals for any fragment could easily be submitted "by hand" following an example (i.e. `--command="--geometry X --procModifiers dd4hep"` in the matrix arguments). Also, dd4hep is now the default for Run 3, so the workflows are superfluous. I can revert this specific change if really necessary, but in general this is not the intent for special workflows.

Reply:
This may have been the case in the past but it's not really like this anymore. And there are many examples here pointing in that direction. I think the advantage of having a wider selection of ready-to-use workflows overcomes any possible drawback here (e.g. a slightly longer time to generate the matrix). Then I agree for Run3 is superfluous. But for Phase2 it is not. So we could simply drop Phase 1 support, but please keep the full workflow for `2026`, thanks.

Reply:
Done in 41125564b1dcb3c15f7fbc9e5deeade5ce297fbb